src/core/blueprint/blueprint.rs

```diff
@@ -262,3 +263,73 @@ impl Blueprint {
         schema.finish().unwrap()
     }
 }
+
+#[cfg(test)]
+mod tests {
```

Comment:
do you mean, I should take snapshots of the blueprint that is generated from config defined in .md files?

Reply:
I tried adding blueprint as snapshot but I think since `BTreeMap` is being used in the type, the order keeps changing and new snapshot gets created everytime

Reply:
no, no, my suggestion is to not focus on blueprint structure at all and focus on the fact that default values should work properly when they are defined. I.e. in test when making query without providing value verify that request hits the mock for default value, and if we do provide some value in query than another mock should be hit

Reply:
Apparently, the same issue that was encountered in #1939 for default value in field arguments, seems to be affecting default values in input types. It will not be possible to test this change by running a query until that issue with `async-graphql` is fixed

Reply:
Delete these tests perhaps. I think @meskill Is correct we should implement them as integration tests. We can mark them as skipped so that with our new GraphQL engine we should be able to execute it.

Reply:
@tusharmath @meskill added a test and marked it as skipped. The added test will pass once the support for default value is added for input types